### DISCUSSION TITLE
Why cant I exclude the xfce screenshot tool from background blur?

### DISCUSSION THREAD

**Discussion by ProboscisMonke** [yshui/picom#889] [Q&A]
When taking screenshots using the default tool for xfce4 picom's blur effect takes hold over the entire screen making it impossible to actually take a screenshot (this issue also persists when using other applications such as BetterCrewLink)

I attempted to remedy the issue (to no avail) by adding the name of the application used to "`Exclude conditions for background blur`"
Snippet of `picom.conf` file below
```# Exclude conditions for background blur.
blur-background-exclude = [
  #"window_type = 'dock'",
  #"window_type = 'desktop'",
  "class_g ?= 'plank' && window_type = 'dock'",
  "class_g = 'Conky'&& window_type = 'desktop'",
  "class_g = 'LibreWolf'",
  "class_g = 'xfce4-screenshooter'",
  "class_g = 'xfce-goodies'",
  "_GTK_FRAME_EXTENTS@:c"

**Comment by Monsterovich** [comment]
```
"name = 'xfce4-screenshooter'",
```

This works for me.

**Comment by ahmubashshir** [comment]
```python
"""
  name = 'xfce4-screenshooter' &&
  class_g = 'Xfce4-screenshooter' &&
  window_type = 'dialog' &&
  ! (
    _NET_WM_ALLOWED_ACTIONS@:a ||
    _NET_WM_STATE@:a ||
    _NET_WM_DESKTOP@:c
  )
"""
```

use this in blur-background-exclude
